eng/tools/spec-gen-sdk-runner/src/types.ts

```diff
@@ -17,6 +17,8 @@ export interface SpecGenSdkCmdInput {
   specRepoHttpsUrl: string;
   headRepoHttpsUrl?: string;
   headBranch?: string;
+  stagedArtifactsFolder?: string;
```

Comment:
These two new properties are output info of `spec-gen-sdk` tool and we should not add them to this type. `SpecGenSdkCmdInput` is the input parameters for `spec-gen-sdk` tool.